packages/db/src/types/execution.ts

```diff
@@ -13,4 +23,4 @@ export interface PostgresExecutionRecord extends PostgresExecution {
   cumulus_id: number,
   created_at: Date,
   updated_at: Date,
-}
+}
```

Comment:
Newline

Reply:
Ah yup! So that reminds me of a workflow question I had: how/when do you lint? Or actually, what is your pre-commit routine like? Do you run `npm run eslint`, address errors/warnings, and then run `npm test` to run the full test suite? Or do you just use VSCode's linting as you go? 

I missed this (and another lint error that I just fixed/pushed) because I forgot to lint the full project before pushing. I'm wondering if it would be valuable to have the lint step as part of `npm run test` or if I just need to adjust my routine 😄